common/persistence/sql/sqlplugin/tests/visibility.go

```diff
@@ -303,11 +303,11 @@ func (s *visibilitySuite) TestDeleteSelect() {
 	s.NoError(err)
 	s.Equal(0, int(rowsAffected))
 
-	selectFilter := sqlplugin.VisibilitySelectFilter{
+	getFilter := sqlplugin.VisibilityGetFilter{
```

Comment:
Why are we using Get instead of Select here, now? Also, if we are doing that, we should rename the tests accordingly.

Reply:
Hm, I could just leave unchanged. SelectFilter with only namespace and run id will do a Get anyway under the hood.

Reply:
Actually, now I remember why I changed.
It's because in the new code, I made it not return an error in this case.
Currently, since Select calls Get in this case, it returns an error if the row is not found, which is kind of misleading because it's a Select.
Also, there's no actual use case of Select for the (namespace, run id) in the code, only in these tests.
I'll leave unchanged in this PR, and move to where I actually need this.